scripts/uberspace-setup-app

```diff
@@ -0,0 +1,40 @@
+#!/bin/bash
```

Comment:
This script is only run initially, right?

Reply:
Yes, both `uberspace-setup-postgres` and `uberspace-setup-app` are meant to be run manually, a single time on a fresh Uberspace account. They are responsible for compiling Postgres, setting up db users and daemons.